### ISSUE TITLE
JAWS does not read the tab text when switching between tabs in a group using the arrow keys.

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#2050]
# Bug Report

## Description
JAWS does not read the tab text when switching between tabs in a group using the arrow keys.

## Steps to Reproduce

1. TBD

## Additional Context / Screenshots
Identified as part of functional verification on [this PR](https://github.com/cerner/terra-core/pull/2004#issuecomment-438396868).

## Expected Behavior
JAWS should read tab text when switching between tabs ina group using arrow keys.

## Environment

* Component Name and Version: terra-tabs
* Browser Name and Version: TBD
